build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts

```diff
@@ -19,4 +19,11 @@ repositories {
         name = "Gradle libs"
         url = uri("https://repo.gradle.org/gradle/libs")
     }
+    maven {
+        name = "CHAMP libs"
+        url = uri("https://releases.usethesource.io/maven/")
```

Comment:
Given that we try to only use our own repo during build, this most likely needs to be added to our CI mirrors.
And a mirroring setup to be done

Reply:
@blindpirate suggested adding the repo.